### ISSUE TITLE
[Health 10.1.1] Code formatting on Android is questionable, can I fix it?

### ISSUE THREAD

**Issue by ciriousjoker** [cph-cachet/flutter-plugins#936]
### Description

There is absolutely no way that 152 spaces of indentation is considered good practice by anyone.
Additionally, the indentation arbitrarily changes to an even higher value past a certain point, why is that?

We're planning to use this at our company and I intend to make some contributions towards stability on Android, but I'm not going to be able to replicate whatever formatting is going on here since Android Studio pushes me towards the standard 4 space indentation even when I disable autoformatting in this repo.

If the current formatting is intentional and not just negligence, I'm very curious to hear the reasons behind it, otherwise:

**TLDR: Are you guys open to a PR that just formats the whole `HealthPlugin.kt` file using the default Kotlin formatter?**

### Additional information

![CleanShot 2024-03-30 at 16 09 22@2x](https://github.com/cph-cachet/flutter-plugins/assets/16866223/c99c55d4-544f-447f-b0d6-32f308d05963)

![CleanShot 2024-03-30 at 16 11 21@2x](https://github.com/cph-cachet/flutter-plugins/assets/16866223/67eb4d49-a8e7-4287-bebd-7cacca87dc97)

**Comment by SlimShadyIAm** [comment]
This should be better on master now :)
